src/mmw/apps/modeling/tasks.py

```diff
@@ -210,6 +244,18 @@ def _get_precip_value(model_input):
         return None
 
 
+def convert_result_areas(pixel_width, results):
+    """
+    Updates area values on the result set from tr55. TR55 gives area as a
```

Comment:
After talking it seem that the comment is misleading but the results are accurate. I'll update these and rebase.